underscore.js

```diff
@@ -1125,6 +1125,8 @@
     }
     // Assume equality for cyclic structures. The algorithm for detecting cyclic
     // structures is adapted from ES 5.1 section 15.12.3, abstract operation `JO`.
+    aStack || (aStack = []);
```

Comment:
This is actually [a](https://github.com/jashkenas/backbone/blob/master/backbone.js#L85-L86) [common](https://github.com/jashkenas/backbone/blob/master/backbone.js#L174-L175) [pattern](https://github.com/jashkenas/backbone/blob/master/backbone.js#L283) [in](https://github.com/jashkenas/backbone/blob/master/backbone.js#L358) [Backbone](https://github.com/jashkenas/backbone/blob/master/backbone.js#L446).